pkg/netpol/eval/internal/k8s/representative_selectors.go

```diff
@@ -53,30 +60,30 @@ func SelectorsFullMatch(ruleSelector, repSelector *v1.LabelSelector) (bool, erro
 	// v1.LabelSelectorAsSelector:
 	// https://github.com/kubernetes/apimachinery/blob/dc7e034c86479d49be4b0eefad307621e10caa0e/pkg/apis/meta/v1/helpers.go#L34
 	// Requirements.Add : https://github.com/kubernetes/apimachinery/blob/d7e1c5311169d5ece2db0ae0118066859aa6f7d8/pkg/labels/selector.go#L373
-	requirements1, _ := selector1.Requirements() // sorted
-	requirements2, _ := selector2.Requirements() // sorted
-	if len(requirements1) != len(requirements2) {
+	ruleRequirements, _ := ruleSelectorConverted.Requirements() // sorted
+	repRequirements, _ := repSelectorConverted.Requirements()   // sorted
+	if len(ruleRequirements) != len(repRequirements) {
 		return false, nil
 	}
-	for i := range requirements1 {
-		// requirements1[i].Equal(requirements2[i]) returns false if the values are not in same order
+	for i := range ruleRequirements {
+		// ruleRequirements[i].Equal(repRequirements[i]) returns false if the values are not in same order
 		// (stringslices.Equal returns true only if two slices have same length and same order of items)
 		// however, `Requirement.String` sorts the values using `safeSort`, so will compare by string
 		// link to Requirement.String() :
 		// https://github.com/kubernetes/apimachinery/blob/d7e1c5311169d5ece2db0ae0118066859aa6f7d8/pkg/labels/selector.go#L310
-		str1 := requirements1[i].String()
-		str2 := requirements2[i].String()
+		str1 := ruleRequirements[i].String()
```

Comment:
change also str1, str2, to follow same convention

Reply:
done